fe/fe-core/src/main/java/com/starrocks/catalog/HiveTable.java

```diff
@@ -371,10 +373,30 @@ public void readFields(DataInput in) throws IOException {
 
     @Override
     public void onCreate() {
+        if (!Config.enable_hms_events_incremental_sync) {
```

Comment:
you should add oncreate operator when load table from image. ref: LocalMetastore.loadDb();